### ISSUE TITLE
OpenTracing Span being lost in calls to NSMgr

### ISSUE THREAD

**Issue by edwarnicke** [networkservicemesh/networkservicemesh#1374]
## Expected Behavior

It is expected that all clients should be sending OpenTracing span ids and they should be propogated via context for all calls made by the components in NSM(NSMgr, NSM Forwarder, etc).  

## Current Behavior

In recent experiments, the NSMgr shows many traces with no parent span, indicating that NSMgr is dropping client context somewhere and thus not propogating spans.

## Failure Information (for bugs)

Run NSM examples like vpp-icmp and look at the jaeger ui to see spans.

## Steps to Reproduce

Run NSM examples like vpp-icmp and look at the jaeger ui to see spans.
## Context

* Kubernetes Version:
* etc.

**Comment by tiswanso** [comment]
@edwarnicke I believe #1359 would have fixed this if you setup the env vars for Jaeger in the nsm-admission-webhook pod:

In #1368 I'm adding those vars to the helm chart:
https://github.com/networkservicemesh/networkservicemesh/pull/1368/files#diff-71c1a6beac8683aeb9c0ef25a98f5c40R38

**Comment by onobrod** [comment]
Tried to reproduce the issue three times: on the latest master, on one week old master (just after #1337 has been merged) and on two weeks old master (before #1337 and #1359 have been merged) and got the same results — only one '<trace-without-root-span>' in nsmd@nsmgr service, other services seems good.

To reproduce I added jaeger envs into all pods (just like it is done in #1368) and deployed infra and icmp example. @edwarnicke does it similar to what you have seen? Should we try to fix this one broken trace or is there a way to reproduce an issue with lots of these traces?
